dice-game/README.md

```diff
@@ -0,0 +1 @@
+# dice-game
```

Comment:
could you do a bit of a write up here, if only to link to the docs? [r-p-s](https://github.com/oasislabs/tutorials/tree/master/rock-paper-scissors) is not spec compliant (which you might consider rectifying, too :)

Reply:
Currently implementing a link to https://docs.oasis.dev/, will update this to  a more granular link if/when a tutorial/relevant docs are created for this

Reply:
also: https://github.com/oasislabs/tutorials/pull/58